packages/p/plasma-welcome/package.yml

```diff
@@ -0,0 +1,38 @@
+name       : plasma-welcome
+version    : 6.1.5
+release    : 1
+source     :
+    - https://download.kde.org/stable/plasma/6.1.5/plasma-welcome-6.1.5.tar.xz : 12ece9e7d61c3ec6a23fe9357667416e26802abafdc69f245cc3d96a05ef9d96
+homepage   : https://invent.kde.org/plasma/plasma-welcome
+license    : BSD-3-Clause and GPL-2.0-or-later
```

Comment:
We do this as a list:
```
- BSD-3-Clause
- GPL-2.0-or-later
```

The checks don't understand the lists, but that's fine

Reply:
I took it directly from Fedora and they use a single line haha oops. Honestly the licensing stuff is confusing and even Nate who is the main developer of Plasma Welcome is partially confused by it.

If anyone is skilled at licensing stuff please verify I have got it right.